kubeflow/gcp/iap.libsonnet

```diff
@@ -385,6 +386,10 @@
                     name: "SERVICE",
                     value: if params.useIstio then "istio-ingressgateway" else "envoy",
                   },
+                  {
+                    name: "INGRESS_SERVICE",
```

Comment:
same here, the env var should be INGRESS or INGRESS_NAME

Reply:
done.